src/Window.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useReducer, useCallback } from "react";
```

Comment:
```suggestion
import React, { useState, useReducer } from "react";
```
This will fix your code smell.